OMCompiler/Compiler/Template/CodegenC.tpl

```diff
@@ -2857,8 +2857,8 @@ template generateStaticSparseData(String indexName, String systemType, list<tupl
   case _ then
       let sp_size_index =  lengthListElements(unzipSecond(sparsepattern))
       let sizeleadindex = listLength(sparsepattern)
-      let colPtr = genSPCRSPtr(listLength(sparsepattern), sparsepattern, "colPtrIndex")
```

Comment:
In case you are wondering what is happening here: Susan does not understand this :(. `let sizeleadindex = ..` will have `Tpl.Text` type instead of `Integer`. Basically it is converted to string instead of being an Integer. The error messages from Susan can be confusing sometimes.

Reply:
@mahge, that is what I tried to say... 🚀

Reply:
Ya I saw the notification :). I was already writing it so added it anyway.

Reply:
> The template language allows local definitions that can be referred to in the template body. A local definition is bound to a text value.

I'm sorry for not seeing that error. Thank you for the explanation!